src/GridFieldOrderableRows.php

```diff
@@ -587,53 +588,20 @@ protected function reorderItems($list, array $values, array $sortedIDs)
         // sanity check.
         $this->validateSortField($list);
 
-        $isVersioned = false;
-        // check if sort column is present on the model provided by dataClass() and if it's versioned
-        // cases:
-        // Model has sort column and is versioned - handle as versioned
-        // Model has sort column and is NOT versioned - handle as NOT versioned
-        // Model doesn't have sort column because sort column is on ManyManyList - handle as NOT versioned
-        // Model doesn't have sort column because sort column is on ManyManyThroughList - inspect through object
-        if ($list instanceof ManyManyThroughList) {
-            // We'll be updating the join class, not the relation class.
-            $class = $this->getManyManyInspector($list)->getJoinClass();
-            $isVersioned = $class::create()->hasExtension(Versioned::class);
-        } elseif (!$this->isManyMany($list)) {
-            $isVersioned = $class::create()->hasExtension(Versioned::class);
-        }
-
-        // Loop through each item, and update the sort values which do not
-        // match to order the objects.
-        if (!$isVersioned) {
+        // ManyManyList models are not represented by the ORM, and so they need to be updated through an SQL Query
```

Comment:
> ManyManyList models are not represented by the ORM

`ManyManyList` is literally the ORM representation for this relation type... did you mean the relation itself doesn't have a `DataObject` backing it up (i.e. the `sort` is on a table that doesn't have a `DataObject` representation)?

If so, maybe a better way of wording this is "ManyManyList extra fields aren't easily updated via the ORM, and so they need to be updated through an SQL Query"

Also now that I've noticed that this is feature missing in the ORM I'm going to see if I can quickly whip up a PR to add it in. Seems like an obvious thing to want.

Reply:
> ManyManyList is literally the ORM representation for this relation type... did you mean the relation itself doesn't have a DataObject backing it up (i.e. the sort is on a table that doesn't have a DataObject representation)?

Yup, that's exactly what I meant.

Thanks for the suggestion 😃 I'll update the comment.

Reply:
FYI that PR is up: https://github.com/silverstripe/silverstripe-framework/pull/10406
Now I can get back to reviewing this one 😅